src/applications/yellow-ribbon/components/SearchResult/index.jsx

```diff
@@ -0,0 +1,84 @@
+// Node modules.
+import React from 'react';
+import PropTypes from 'prop-types';
+import { invoke } from 'lodash';
+// Relative imports.
+import { capitalize } from '../../helpers';
+
+const SearchResult = ({ school }) => (
+  <div className="medium-screen:vads-l-col vads-l-col vads-u-background-color--gray-light-alt vads-u-margin-bottom--2 vads-u-padding-x--3 vads-u-padding-y--2">
+    {/* School Name */}
+    <h3 className="vads-u-margin--0">{capitalize(school?.name)}</h3>
+
+    {/* School Location */}
+    <p className="vads-u-margin-bottom--1 vads-u-margin-top--0">
+      {capitalize(school?.city)}, {school?.state}
+    </p>
+
+    <div className="vads-l-row vads-u-margin-top--2">
+      <div className="vads-l-col--6 vads-u-display--flex vads-u-flex-direction--column vads-u-justify-content--space-between">
+        {/* Max Benefit Amount */}
+        <div className="vads-u-col">
+          <h4 className="vads-u-font-family--sans vads-u-font-size--h5 vads-u-margin--0">
+            Maximum Yellow Ribbon benefit amount
+          </h4>
+          <p className="vads-u-margin--0">
+            {invoke(school?.tuitionOutOfState, 'toLocaleString', 'en-US', {
+              currency: 'USD',
+              style: 'currency',
+            }) || 'Unknown'}
```

Comment:
I like this solution. This could be a useful platform function, to help make it more discoverable if other teams run into it.